### PR TITLE
Tests for task requests

### PR DESCRIPTION
**Pull request by TanishqSingla**

### CODE REVIEW

File: __tests__/taskRequests/taskRequest.test.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  describe('When the user is super user', () => {
+    it('should display the task requests card', async () => {
+      const url = await page.evaluate(() => API_BASE_URL);
+      console.log(url);
```
Comment (prakashchoudhary07) [inline __tests__/taskRequests/taskRequest.test.js:47]: Please remove console.log

File: __tests__/taskRequests/taskRequest.test.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      const taskCards = await page.$$('.taskRequest__card');
+      const title = await taskCards[0].evaluate(
+        (el) => el.children[0].textContent,
+      );
+      const purpose = await taskCards[0].evaluate(
+        (el) => el.children[1].textContent,
+      );
```
Comment (prakashchoudhary07) [inline __tests__/taskRequests/taskRequest.test.js:54]: NIT: By accessing children in this way, later if the HTML structure changes then the test will fail

File: __tests__/taskRequests/taskRequest.test.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    await page.goto(`${SITE_URL}/taskRequests`);
+    await page.waitForNetworkIdle();
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  describe('When the user is super user', () => {
+    it('should display the task requests card', async () => {
```
Comment (prakashchoudhary07) [inline __tests__/taskRequests/taskRequest.test.js:45]: there should have been a test, to check if user is not super user

**Review by prakashchoudhary07 — CHANGES_REQUESTED** [review]

File: __tests__/taskRequests/taskRequest.test.js (hunk 2 above)
Comment (Achintya-Chatterjee) [reply __tests__/taskRequests/taskRequest.test.js:54]: can rewrite the code @prakashchoudhary07

**Comment by Achintya-Chatterjee** [comment]
This PR has been closed because it is moved to a new PR #460

**Pull request merged**